flair/datasets/treebanks.py

```diff
@@ -1226,6 +1226,34 @@ def __init__(self, base_path: Union[str, Path] = None, in_memory: bool = True):
 
         super(UD_NORTH_SAMI, self).__init__(data_folder, in_memory=in_memory)
 
+        
+class UD_MARATHI(UniversalDependenciesCorpus):
+    def __init__(self, base_path: Union[str, Path] = None, in_memory: bool = True):
+
+        if type(base_path) == str:
+            base_path: Path = Path(base_path)
+
+        # this dataset name
+        dataset_name = self.__class__.__name__.lower()
+
+        # default dataset folder is the cache root
+        if not base_path:
+            base_path = Path(flair.cache_root) / "datasets"
+        data_folder = base_path / dataset_name
+
+        # download data if necessary
+        web_path = "https://raw.githubusercontent.com/UniversalDependencies/UD_English-EWT/master"
```

Comment:
These paths still point to the English UD treebank. They should point to the Marathi treebank.

Reply:
Path and file names amended.